governance-exercises/EXERCISE_POLLS_DAOMasons/poll_exercise.md

```diff
@@ -0,0 +1,45 @@
+    - For changes to simple variables like debt ceiling and target available debt, and in the interest of maintaining momentum we think it would be better to get a poll started soon so they can make it into the next round of Executives. 
+    - A case could be made for holding the token onboarding portion until the monthly poll, but because we're breaking this issue into 2 or 3 related polls, holding them all concurrently makes sense as they can be considered as a group without requiring voters to consume a lot of additional contexts 
+- We saw a couple of ways we could break this down into multiple polls. Our first instinct was to gauge from the templates and historical examples of similar polls to do the following: 
+    1) Debt Ceiling, Target Available Debt & Debt Cooldown increase. 
+    2) Increase the DAI deposit rate on Spork from 2 to 2.5% 
+    3) Onboard ABC Token. 
+- However, we believe it makes more sense to bundle 1 & 2 as increasing the incentive to deposit without increasing the debt ceiling might lead to a stall and stunt growth which is the opposite of the original poster's intention. 
+- During this process, we will stay up to date with any agreed-upon changes and modify polls accordingly. 
+- If there is a debate, ensure that it comes to a natural end before finalizing the polls.
+- Determine Impact Estimation
+        - As per [Maker Operation Manual](https://manual.makerdao.com/) section on Off-Chain Governance, [Impact Estimations](https://manual.makerdao.com/governance/off-chain/impact-estimations) ```GovAlpha applies impact estimations to active governance items (MIPs and Signal Requests). The goal of these estimations is to provide delegates, core unit members, and the wider community with an idea of how important it is that they pay attention to a given governance item given limited time.```
+        - Also provides a guideline to be used for creating an estimation.
+            - *In this case, adjusting the D3M is likely to be considered high risk* "Anything that adds or removes concretely identifiable risk to the Maker Protocol is likely to be marked as high impact."
+            - Onboarding ABC would likely be considered Medium Risk, as Stability Scope has mentioned they have no concerns about it.
+            - Changing the DAI deposit rate is likely a higher risk, as it is designed to provide for rapid growth, with rapid growth comes higher risk. I would also think that any changes to "percentage rates" are high risk.
+```If there is any doubt about a given estimation, it is generally rounded up.```
+- Once the discussion has reached its natural end, push the polls to the [X repo] so it can undergo the [x process]. 
+
+### Draft any forum responses required or implied by your course of action.
+
+- As the forum post currently stands, we would not provide a response.
+- However, we would hit a 'like' or heart on the post to indicate that we have seen it and are aware of it.
```

Comment:
Ah, actually I missed something here. Probably worth communicating that you're doing two polls rather than the one poll that was being assumed on the thread thus far. Just to ensure authors + authorizers have time to complain or comment if they want to.